react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class NewsLetterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: ' ' };
```

Comment:
Starting a form control's value with a space (`' '`) in it means a) the user has to delete the space, and b) they can't see a placeholder until they do. Use an empty string (`''`) instead.